src/otg/clump.py

```diff
@@ -63,7 +63,7 @@ def __post_init__(self: ClumpStep) -> None:
         else:
             sumstats = SummaryStatistics.from_parquet(
                 self.session, self.input_path, recursiveFileLookup=True
-            )
+            ).coalesce(400)
```

Comment:
Number estimate on the basis of 60 workers x 16 cores x 4 partitions
```suggestion
            ).coalesce(4000)
```